statler_site/views.py

```diff
@@ -9,7 +9,8 @@ def healthCheck(request):
 def getPlayListPage(request):
     """returns an html page which will use JavaScript
     to get and display data"""
-    return render_to_response('play-list-page.html')
+    # 'all'  is the hardcoded play list name for getting all plays
```

Comment:
Perhaps, but it needs the ranking, which I expected to live in a list (though one created at the api level, not one that lives in the database). Also, this doesn't allow real data and test data to coexist. I think it would be cleaner to have a real list (maybe called all) that includes all the plays that we ever want a user to actually see.